python/unitytrainers/models.py

```diff
@@ -44,109 +56,137 @@ def create_visual_input(o_size_h, o_size_w, bw, name):
         visual_in = tf.placeholder(shape=[None, o_size_h, o_size_w, c_channels], dtype=tf.float32, name=name)
         return visual_in
 
-    def create_vector_input(self, s_size):
+    def create_vector_input(self):
```

Comment:
Should `create_vector_input` take in a name as argument in the same way `create_visual_input` does ?